### ISSUE TITLE
Microsoft SQL Server Support

### ISSUE THREAD

**Issue by ccgintranet** [prooph/pdo-event-store#214]
Hello,

Does this library support Microsoft SQL Server instances?

Thanks,

Regards,

**Comment by prolic** [comment]
No it does not.

**Comment by codeliner** [comment]
Hey @ccgintranet ,
sorry, we don't support Microsoft SQL Server instances. We gave up on the idea supporting all DB vendors to be able to optimize for advanced use cases like projections.

**Comment by chad-w-cvt** [comment]
Hi @ccgintranet,

Early on when our company decided to start using prooph components we were restricted to only use Sql Server so I started the process of supporting Sql Server in pdo-event-store.  Here's a link to our repo [https://github.com/cvtrust/pdo-event-store](https://github.com/cvtrust/pdo-event-store).  We actually ended up being able to use Postgres but the code was pretty close to working, but I'm sure is way out of date at this point, but would be a good starting point.  What version of Sql server are you using?

Chad
